src/main/resources/templates/projects/discuss.ftlh

```diff
@@ -1,32 +1,33 @@
 <#import "/spring.ftl" as spring />
 <#import "*/utils/hangar.ftlh" as hangar />
+<#import "*/projects/view.ftlh" as view/>
+<#import "*/utils/editor.ftlh" as editor/>
 
-@*
+<#--
 Discussion page within Project overview.
-*@
-@import controllers.sugar.Requests.OreRequest
-@import models.viewhelper.{ProjectData, ScopedProjectData}
-@import ore.OreConfig
-@import ore.markdown.MarkdownRenderer
-@import views.html.helper.CSPNonce
-@import views.html.utils
+-->
+<#--@import controllers.sugar.Requests.OreRequest-->
+<#--@import models.viewhelper.{ProjectData, ScopedProjectData}-->
+<#--@import ore.OreConfig-->
+<#--@import ore.markdown.MarkdownRenderer-->
+<#--@import views.html.helper.CSPNonce-->
+<#--@import views.html.utils-->
```

Comment:
Not like this is hurting in this instance, but commenting out the entire block vs. every single line would probably be nicer :p

Reply:
```suggestion
<#--@import controllers.sugar.Requests.OreRequest
@import models.viewhelper.{ProjectData, ScopedProjectData}
@import ore.OreConfig
@import ore.markdown.MarkdownRenderer
@import views.html.helper.CSPNonce
@import views.html.utils-->
```